modules/base/admin/module.py

```diff
@@ -451,6 +451,22 @@ async def config_set(self, ctx, key: str, value: str):
         if key in ("prefix", "status"):
             await utils.Discord.update_presence(self.bot)
 
+    @commands.check(acl.check)
+    @commands.group(name="pumpkin")
+    async def pumpkin_(self, ctx):
+        await utils.Discord.send_help(ctx)
+
+    @commands.check(acl.check)
+    @pumpkin_.command(name="restart")
+    async def pumpkin_restart(self, ctx):
+        """This won't work without system-level error detection."""
+        exit(1)
```

Comment:
Should work fine but maybe we can include a start and restart scripts where restart would handle the termination of running instance and spawn a new one. I'll say that's up for discussion later. Accepting the PR